### ISSUE TITLE
Poor Result

### ISSUE THREAD

**Issue by sadichel** [HasnainRaz/Fast-AgingGAN#17]
On training the model on UTKFaces to compare the result with the pretrained model for 100 and 200 epochs, I got a loss of about 1.67 for both epochs. While using the trained weights (see model.load_state_dict(ckpt, strict=False), i have to set strict to false if not, i get error when loading it.
I am getting gray color image when I infer any image with the trained model and I don't know why. Pls, where do you think my problems are in the model?

**Comment by HasnainRaz** [comment]
I would be able to help more with the grayscale images if you can provide the script you are using to infer with the trained model.

The reason you can't load the state dict without the "strict=False" is because the weights are for the entire GAN model (generator + discriminator), but you only need the generator weights in the infer script. You can simply separate the generator weights from the state dict and load those (this was also done with the provided pretrained weights).

**Comment by HasnainRaz** [comment]
Closing due to inactivity
